src/PostRector/Rector/UnusedImportRemovingPostRector.php

```diff
@@ -55,25 +55,31 @@ public function enterNode(Node $node): ?Node
                 continue;
             }
 
-            $useUse = $namespaceStmt->uses[0];
             $isCaseSensitive = $namespaceStmt->type === Use_::TYPE_CONSTANT;
 
-            if ($isCaseSensitive) {
-                $names = $namesInOriginalCase;
-            } else {
-                if ($namesInLowerCase === null) {
-                    $namesInLowerCase = array_map(strtolower(...), $namesInOriginalCase);
+            $useCount = count($namespaceStmt->uses);
+            foreach ($namespaceStmt->uses as $useUseKey => $useUse) {
+                if ($isCaseSensitive) {
+                    $names = $namesInOriginalCase;
+                } else {
+                    if ($namesInLowerCase === null) {
```

Comment:
this lazy load check seems make harder to read, and use only constant is rarely used, just move variable initialization above loop :)

Reply:
Sure. Removed.